lib/aruba/environment/basic_environment.rb

```diff
@@ -0,0 +1,129 @@
+module Aruba
```

Comment:
Sorry, for not being more clear:

PosixEnvironment -> Environments::PosixEnvironment
BasicEnvironment -> Environments::BasicEnvironment